src/data/faq.json

```diff
@@ -352,19 +352,54 @@
-                            "<b>Why is the 2G+ status not displayed even though I have a vaccination or recovery certificate and a negative test?</b>",
-                            "The 2G+ status results from the combination of vaccination and test certificates. If the data within the certificates, for example name or date of birth, do not match, the Corona-Warn-App cannot assign these certificates to each other and determine the corresponding status. In this case, the Corona-Warn-App will show you two certificates in the overview, each of which is assigned the rules 3G and 2G. Therefore, make sure that your name is entered correctly and completely when issuing the certificate, and contact the issuer of the certificate if errors occured when entering the name or date of birth.",
-                            "A test certificate officially confirms a negative test result in the form of a QR code in the Corona-Warn-App. You can request the official test certificate for PCR and rapid tests when you register a test in the Corona-Warn-App. To do this, you can open the universal QR code scanner in the middle of the CWA tab and scan the QR code for test registration. You will receive this either while booking the test or on-site during testing, provided the provider is connected to the Corona-Warn-App. After scanning the QR code, a window will automatically open where you can request the test certificate. More information can be found in this blog: <a href='../blog/2021-06-24-cwa-version-2-4' target='_blank'>Project team releases rapid test partner search and Corona-Warn-App version 2.4 with digital test certificate</a>",
-                            "<b>How do I use the combined 2G+ display?</b>",
-                            "If you have stored a valid vaccination or recovery certificate and a valid test certificate in the app, these will also be shown in a combined display from version 2.16 onwards. You can then use the switch to switch between the QR code of the '2G certificate' (vaccination or recovery certificate) and that of the test certificate to quickly and easily provide 2G+ proof. Both QR codes must be checked on site with the CovPassCheck app together with presentation of the ID card if 2G+ proof is required. For privacy reasons, when the certificates are checked by the CovPassCheck app, it is not apparent whether the individual has a vaccination or recovery certificate. When scanning the QR codes, the checkers only see whether the certificate is valid or not."
+                            "<b>Updated on February 23, 2022:</b> Starting with version 2.18, the Corona-Warn-App will also show you the 2G+ status proof for 'freshly' vaccinated, 'freshly' recovered, and 'boostered' users if the corresponding COVID digital certificates are stored in the app and also have been issued to the same name as well as date of birth.",

```

Comment:
"to the same name" => "for the same name"

Reply:
Fixed in [5ea7690](https://github.com/corona-warn-app/cwa-website/pull/2461/commits/5ea76904b79666aaa74d6c4dee7b9bab85147183), thanks!